ubuntudesign/docs.vanillaframework.io
# Add favicon link to head

## Done

Add a favicon ref to <head> of template file

## QA

- Sanity check code
- Run local docs import once https://github.com/vanilla-framework/vanilla-framework/pull/1023 has been released.
